packages/design-system-tokens/src/tokens/Theme.cmsgov.json

```diff
@@ -4507,7 +4507,7 @@
       },
       "trigger": {
         "color": {
-          "$value": "{theme.color.primary}",
+          "$value": "{theme.color.black}",
```

Comment:
To confirm, did you change these directly in JSON or make the change in Figma and then sync to code?

Reply:
Changed directly in JSON. I didn't sync anything yet.